lib/apiservers/portlayer/restapi/handlers/misc_handlers.go

```diff
@@ -29,28 +28,28 @@ import (
 type MiscHandlersImpl struct{}
 
 // Configure assigns functions to all the miscellaneous api handlers
-func (handler *MiscHandlersImpl) Configure(api *operations.PortLayerAPI, handlerCtx *HandlerContext) {
-	api.MiscPingHandler = misc.PingHandlerFunc(handler.Ping)
-	api.MiscGetVCHInfoHandler = misc.GetVCHInfoHandlerFunc(handler.GetVCHInfo)
+func (h *MiscHandlersImpl) Configure(api *operations.PortLayerAPI, handlerCtx *HandlerContext) {
+	api.MiscPingHandler = misc.PingHandlerFunc(h.Ping)
+	api.MiscGetVCHInfoHandler = misc.GetVCHInfoHandlerFunc(h.GetVCHInfo)
 }
 
 // Ping sends an OK response to let the client know the server is up
-func (handler *MiscHandlersImpl) Ping() middleware.Responder {
+func (h *MiscHandlersImpl) Ping(param misc.PingParams) middleware.Responder {
```

Comment:
Parameter-less operations in the swagger spec now causes parameters to be generated?